src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -1144,17 +1126,6 @@ private void onApproveUserAccess() {
 		getApproveUserAccessModal().show();
 	}
 
-	private void onAddEvaluationQueue() {
-		getEvaluationEditorModal().configure(entity.getId(), new Callback() {
-			@Override
-			public void invoke() {
-				Place gotoPlace = new Synapse(entity.getId(), null, EntityArea.CHALLENGE, null);

```

Comment:
this got replaced w/ a no-op dummy callback in ChallengeTab since there's no interaction that would allow you to leave the `CHALLENGE` Place anyways once the modal is shown.